client/src/test/java/com/paypal/selion/android/sample/AppiumAndroidScrollTest.java

```diff
@@ -32,9 +32,9 @@
 
     private static final String pageObjectsAppPath = "src/test/resources/apps/PageObjectsDemoApp.apk";
     private static final String deviceName = "android:19";
-    private String messageBoxLocator = "android:id/message";
-    private String menuLocator ="com.paypal.selion.pageobjectsdemoapp:id/action_button";
-    private String listViewLocator = "com.paypal.selion.pageobjectsdemoapp:id/scroll_view";
+    private static final String messageBoxLocator = "android:id/message";
+    private static final String menuLocator ="com.paypal.selion.pageobjectsdemoapp:id/action_button";
+    private static final String listViewLocator = "com.paypal.selion.pageobjectsdemoapp:id/scroll_view";
```

Comment:
This field and others like it in this file should now have an all uppercase name.